### ISSUE TITLE
Allow other types of tables in panels

### ISSUE THREAD

**Issue by danielfriis** [kaspernj/bootstrap_builders#1]
Would be great to have the option to use table types other than `table-striped`.

**Comment by kaspernj** [comment]
What do you think about this:
`= bb_table :striped, :hover, :bordered do`

Instead of:
`= bb_table do`

Panels with tables like this:
`= bb_panel table: [:striped, :hover, :bordered] do`

Instead of:
`= bb_panel table: true do`

**Comment by danielfriis** [comment]
@kaspernj - That would be awesome!

**Comment by kaspernj** [comment]
@danielfriis Please check the following PR:
https://github.com/kaspernj/bootstrap_builders/pull/3

Does that solve the problem?

**Comment by danielfriis** [comment]
@kaspernj - Look's good. I see what your trying to do, by having *one central* place to define the tables of the application. However, I actually liked the syntax of your initial idea better:

```
= bb_panel table: [:striped, :hover] do
```

Then in the `add_table` action you could do:

```
classes = @table.map { |class| "table-#{class}" }
```

What do you think?

**Comment by kaspernj** [comment]
I recall I was afraid, that it wouldn't be possible to add a class, that wasn't a "table-*"-class, but I guess those could be added through the `:class` argument, now that I think more about it.

**Comment by danielfriis** [comment]
@kaspernj - What's the verdict then? :)

**Comment by kaspernj** [comment]
@danielfriis I have fixed it in ab9c743 :-)
